tests/e2e/controlplane/control_plane_test.go

```diff
@@ -168,17 +171,97 @@ spec:
 					})
 				})
 
+				When("Sail Operator configuration", func() {
+					BeforeAll(func() {
+						Expect(configconverter.SetEnvVariables("ISTIO_VERSION", version.Name)).To(Succeed(), "error adding env variable: ISTIO_VERSION")
+						Expect(configconverter.SetEnvVariables("ISTIO_NAMESPACE", controlPlaneNamespace)).To(Succeed(), "error adding env variable: ISTIO_NAMESPACE")
+						istioYamlText := `
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  name: default
+spec:
+  components:
+    pilot:
+      enabled: true
+      k8s:
+        resources:
+          requests:
+            cpu: 1000m # override from default 500m
+            memory: 4096Mi # ... default 2048Mi
+        hpaSpec:
+          maxReplicas: 10 # ... default 5
+          minReplicas: 2  # ... default 1
+        nodeSelector: # ... default empty
+          master: "true"
+        tolerations: # ... default empty
+        - key: dedicated
+          operator: Exists
+          effect: NoSchedule
+        - key: CriticalAddonsOnly
+          operator: Exists
+  values:
+    global:
+      istiod:
+        enableAnalysis: true
+    pilot:
+      env:
+        PILOT_ENABLE_STATUS: true`
+						Log("Istio in-cluster Operator Config YAML:", indent(2, istioYamlText))
+						istioYamlFileWithPath, err = configconverter.SaveYamlToFile(istioYamlText)
+						Expect(err).To(Succeed(), "failed to write YAML file")
+						Success("Provided istio config saved to file")
+					})
+
+					It("is created", func() {
+						Expect(configconverter.ExecuteConfigConverter(istioYamlFileWithPath)).To(Succeed(), "error in execution of ./configuration-converter.sh")
+						Success("Sail Operator Yaml Created")
+					})
+
+					It("is validated", func() {
+						sailYamlText := `
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  values:
+    global:
+      istiod:
+        enableAnalysis: true
+    pilot:
+      env:
+        PILOT_ENABLE_STATUS: "true"
+      enabled: true
+  version: %s
+  namespace: %s`
+						sailYamlText = fmt.Sprintf(sailYamlText, version.Name, controlPlaneNamespace)
+						isConversionValidated, err := configconverter.ValidateYamlContent(sailYamlText)
+						Expect(err).To(Succeed(), "failed to read file")
+						Expect(isConversionValidated).To(BeTrue(), "YAML content is not as wanted")
+						Success("Conversion completed")
+					})
+				})
+
 				When("the Istio CR is created", func() {
 					BeforeAll(func() {
-						istioYAML := `
+						sailYAML := `
 apiVersion: sailoperator.io/v1
 kind: Istio
 metadata:
   name: default
 spec:
+  values:
+    global:
+      istiod:
+        enableAnalysis: true
+    pilot:
+      env:
+        PILOT_ENABLE_STATUS: "true"
+      enabled: true
```

Comment:
Why this change? The purpose of the test was to check if Istio is deployed properly with a minimal `Istio` resource.